scripts/cron-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -vx
+
+# Revert the cache-min setting, since the internal cache does not apply to
+# these repos (and causes problems in lookups)
+npm config set cache-min 10
+
+# Use newer, faster npm
```

Comment:
This version of `npm` is fairly old lets remove this comment

Reply:
Removed. Do you suggest I update the version of `npm` as well?